packages/ui/src/components/va-button/VaButton.vue

```diff
@@ -170,9 +170,29 @@ const {
 } = useButtonBackground(colorComputed, isPressed, isHovered)
 const contentColorComputed = useButtonTextColor(textColorComputed, colorComputed, isPressed, isHovered)
 
+const computedPxSizeStyle = computed(() => {
+  if (typeof props.size === 'number') {
+    return {
+      '--va-button-px-size': `${props.size}px`,
```

Comment:
@Fsss126, please, review this logic and how it can be applied to sizes config.

Reply:
We have a size config refactoring in progress that would provide a more generalized way to solve cases like this one and make and desired customizations in-place https://github.com/epicmaxco/vuestic-ui/pull/4246.

I would suggest releasing that one first and see if a need to pass px values would make sense to support then.